superset/assets/src/visualizations/nvd3/NVD3Vis.js

```diff
@@ -355,12 +356,31 @@ function nvd3Vis(element, props) {
             }
             const xField = formData.granularitySqla;
             const yField = findYAxisField(yColumn, publishedColumns);
-
+            let xFieldVal;
+            let yFieldVal
+            let xValueChanged;
+            let yValueChanged;
```

Comment:
can we explicitly initalize this to false

Reply:
done